test/e2e/backward_compat_test.go

```diff
@@ -1,138 +0,0 @@
-// +build e2e
```

Comment:
Removing this test due to v1 CRDs validation, which does not allow unknown fields to exist in the spec.

Reply:
sounds good!